src/otaclient/create_standby/resume_ota.py

```diff
@@ -123,6 +126,13 @@ def resume_ota(self) -> None:
                     try:
                         expected_digest = bytes.fromhex(_entry_digest_hex)
                     except ValueError:
+                        self._remove_entry(_entry_digest_hex, _resource_dir_fd)
+                        continue
+
+                    # NOTE(20250821): only scan resources that we need, for resources we don't need,
+                    #                 just remove it from the resources dir.
+                    if expected_digest not in self._all_resource_digests:
+                        self._remove_entry(_entry_digest_hex, _resource_dir_fd)
                         continue
```

Comment:
Also, this prevent unneeded resources still remaining at the standby slot(cleanup).